### PR TITLE
Remove telemetry code. Default to benchmarks stored locally.

### LINKED ISSUE #8: VERSION file is missing in repository
VERSION file is missing in repository as result Makefile is missing VERSION variable and build arguments are empty.

### PR DESCRIPTION
**Pull request by chris-wood**
Closes #8 and #9.

### CODE REVIEW

File: commands/commands.go [hunk 1]
```diff
@@ -104,13 +104,26 @@ var Commands = []cli.Command{
...
 			},
+			cli.StringFlag{
+				Name:  "logout",
+				Value: "log.txt",
+			},
 			cli.StringFlag{
 				Name:  "out",
-				Value: "data/data-test.txt",
+				Value: "",
+			},
```
Comment (sudheesh001) [inline commands/commands.go:114]: Do we want to continue maintaining `out` in addition to `logout`?

File: commands/request.go [hunk 2]
```diff
@@ -97,26 +96,6 @@ func resolveObliviousQuery(query odoh.ObliviousDNSMessage, useProxy bool, target
...
-
-	var data DiscoveryServiceResponse
-	decoder := json.NewDecoder(resp.Body)
-	err = decoder.Decode(&data)
-	if err != nil {
-		log.Fatalf("Unable to decode the obtained JSON response from the Discovery service %v\n", err)
-	}
-	return data, nil
-}
-
```
Comment (sudheesh001) [inline commands/request.go:119]: Is there a reason for this? Don't we want the users to be able to bootstrap from publicly available lists?

Comment (chris-wood) [reply commands/request.go:119]: Yeah -- we aren't maintaining that list, so this seems too fragile to support for now. Users of this tool can always build their own list.

File: commands/commands.go (hunk 1 above)
Comment (chris-wood) [reply commands/commands.go:114]: This is the output file for writing the JSON-encoded results, not the log output.

**Review by sudheesh001 — APPROVED** [review]

File: commands/commands.go (hunk 1 above)
Comment (sudheesh001) [reply commands/commands.go:114]: Ah missed that, thank you, this is cleaner!

**Pull request merged**